scripts/start-story/main.go

```diff
@@ -115,6 +120,14 @@ func run() error {
 	}
 	finish()
 
+	if meta.JiraIssue.Fields.Status.Name == "To Do" || meta.JiraIssue.Fields.Status.Name == "Pending" {
```

Comment:
Nit: might want to make these strings constants. We'll probably never change them in JIRA, but if we do, changing one constant is easier than search/replace.